server/src/dto/user.dto.ts

```diff
@@ -56,7 +56,7 @@ export class CreateUserDto {
   source: string;
 
   @IsString()
-  @IsNotEmpty()
+  @IsOptional()
   @Transform(({ value }) => sanitizeInput(value))
   workspaceName: string;
```

Comment:
You can create a separate DTO extending from this CreateUserDto for this purpose. It will be easier to maintain.
```suggestion
  export class OnboardUserDto extends CreateUserDto {
  @IsString()
  @IsOptional()
  @Transform(({ value }) => sanitizeInput(value))
  @MaxLength(500)
  workspaceName: string;
}

```